src/Symfony/Component/Validator/Tests/Constraints/IbanValidatorTest.php

```diff
@@ -173,19 +181,25 @@ public function testInvalidIbans($iban, $code)
     public function getInvalidIbans()
     {
         return array(
-            array('CH93 0076 2011 6238 5295', Iban::CHECKSUM_FAILED_ERROR),
-            array('CH930076201162385295', Iban::CHECKSUM_FAILED_ERROR),
+            array('CH93 0076 2011 6238 5295', Iban::INVALID_FORMAT_ERROR),
+            array('CH930076201162385295', Iban::INVALID_FORMAT_ERROR),
+            array('NO93 8601117 947', Iban::INVALID_FORMAT_ERROR),
+            array('CH930072011623852957', Iban::INVALID_FORMAT_ERROR),
+            array('CY170020 128 0000 0012 0052 7600', Iban::INVALID_FORMAT_ERROR),
+            array('FR76', Iban::INVALID_FORMAT_ERROR),
```

Comment:
I would like to see tests for

* valid IBANs from every country
* invalid IBANs from every country